### ISSUE TITLE
Include warnings from gradle lint

### ISSUE THREAD

**Issue by marmistrz** [fwcd/kotlin-language-server#151]
When developing for Android, there are some useful checks done by `gradle lint`. For instance, if the `onClick` property doesn't exist, a warning will be issued by `gradle lint`. Those warnings are displayed by Android Studio.

It would be great if KLS could also include those warnings.

**Comment by vasiltabakov** [comment]
I don't agree with this suggestion. Android linting issues are mostly "Android stuff" and not "Kotlin stuff".  Even your example (android:onClick) is something in the XML, why should a Kotlin language server notify about issues in XML? After all this is a language server, not a "framework server" or "platform server".
